.evg.yml

```diff
@@ -74,6 +74,41 @@ buildvariants:
 tasks:
   - name: "build"
     commands:
+      # fetch mongosqltranslate libraries for each platform before building
+      # linux arm
+      - func: "fetch libmongosqltranslate"
+        vars:
+          arch: "arm"
+          platform: "linux"
+          lib_prefix: "lib"
+          ext: "so"
+          release_version: "snapshot"
```

Comment:
This is a placeholder for now for testing purposes. I'm not fully understanding the design doc for this. It says when we want to do a release, we'll use a tag that includes the library version -- `v<major>.<minor>.<patch>-libv<major>.<minor>.<patch>`. Are we supposed to parse the mongnosqltranslate library version from that tag? If so, should we default to "snapshot" for patch builds that do not have the tag value set?

Reply:
Yes, we should.
Ideally, parsing the tag should be done in "export variables" and on top of MDBJDBC_VER, we will now now have another variable like MDBTRANSLIB_VER which you can use in this task.
I checked and there is no separate tickets, so you want to the work as part of this ticket. It's also the only place where it is used really, so it makes sense.

Reply:
Done!